src/manifold/include/manifold.h

```diff
@@ -207,7 +207,7 @@ class Manifold {
   Manifold Mirror(glm::vec3) const;
   Manifold Warp(std::function<void(glm::vec3&)>) const;
   Manifold SetProperties(
-      int, std::function<void(glm::vec3, const float*, float*)>) const;
+      int, std::function<void(float*, glm::vec3, const float*)>) const;
```

Comment:
I switched the order here because I forgot to follow my own style of putting output arguments before input arguments.